Modules/Core/ImageFunction/include/itkInterpolateImageFunction.h

```diff
@@ -129,14 +127,26 @@ class ITK_TEMPLATE_EXPORT InterpolateImageFunction:
     return ( static_cast< RealType >( this->GetInputImage()->GetPixel(index) ) );
   }
 
-#if !defined(ITKV4_COMPATIBILITY)
   /** Get the radius required for interpolation.
    *
    * This defines the number of surrounding pixels required to interpolate at
    * a given point.
    */
+  #if defined(ITKV4_COMPATIBILITY)
+  virtual SizeType GetRadius() const
+  {
+    // if ITKv4 compatibility is enabled then set the radius to the
+    // largest by default.
+    const InputImageType* input = this->GetInputImage();
+    if ( !input )
+      {
+      itkExceptionMacro( "Input image required!" );
+      }
+    return input->GetLargestPossibleRegion().GetSize();
+  }
+  #else
   virtual SizeType GetRadius() const = 0;
```

Comment:
I see, so this enforces a derived interpolator to implement `GetRadius` while providing a default  `GetRadius` definition that disables streaming in case of `ITKV4_COMPATIBILITY` except for the already implemented overrides.